addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml

```diff
@@ -21,7 +21,10 @@
 				<options>
 					<option value="0">North America (915MHz)</option>
 					<option value="1">Europe (868MHz)</option>
-					<option value="2">Australia (915MHz)</option>
+					<option value="2">Australia</option>
+					<option value="3">Fine Offset (915MHz)</option>
```

Comment:
These options do not seem to make much sense, at least they do not fit the description, which says that this parameter specifies the band. We now have multiple entries for 915MHz and multiple ones for 868MHz.
Furthermore, at least to me "Fine Offset" doesn't have any helpful meaning (and if somebody asked me what the difference between "Europe" and "Fine Offset" is, I would have to shrug...).
Maybe some more explanation in the documentation could help here :-)